app/src/main/java/org/fossasia/susi/ai/dataclasses/SkillsBasedOnMetrics.kt

```diff
@@ -0,0 +1,11 @@
+package org.fossasia.susi.ai.dataclasses
+
+import org.fossasia.susi.ai.rest.responses.susi.SkillData
+
+/**
+ *
+ * Created by arundhati24 on 12/07/2018.
+ */
+data class SkillsBasedOnMetrics(
+        var metricsList: ArrayList<Pair<String, List<SkillData>?>> = ArrayList()
```

Comment:
Is the String key from Pair dynamic?

Reply:
No, it is not. We receive an entire object of metrics. Then, I assign a String for the metric "rating", another for "usage" and so on. All these four strings have been mentioned in strings.xml.

Reply:
Then there is no need for a list of pairs of string and list of skilldata, these can be hardcoded into object